packages/hosting/src/pages/practices/_practiceId/module1/annexs/annex2/Sheet1.jsx

```diff
@@ -88,6 +93,17 @@ const Sheet1 = ({
   annex2,
   onConfirmSaveSheet1,
 }) => {
+  const BusinessPositionValue = BusinessPosition.find(
```

Comment:
[ ] Nombre de constante debe ser camelcase y generico `businessPosition` and `professionalCareer`
[ ] Retornar todo el objeto
[ ] Usar de esta manera: businessPosition?.label || ""